src/main/java/org/gradle/profiler/ScenarioLoader.java

```diff
@@ -91,6 +93,8 @@ class ScenarioLoader {
     private static final String ANDROID_STUDIO_JVM_ARGS = "studio-jvm-args";
     private static final String ANDROID_STUDIO_IDEA_PROPERTIES = "idea-properties";
     private static final String JVM_ARGS = "jvm-args";
+    private static final String DELETE_FILE = "delete-file";
+    private static final String COPY_FILE = "copy-file";
```

Comment:
Considering that these should be able to take directories, too, I think `delete` and `copy` would be better names, WDYT?

Reply:
I've been rethinking this, and I think it's okay to include `-file`; it's good to disambiguate that this is about the file system.